go/vt/vttablet/tabletserver/txthrottler/tx_throttler.go

```diff
@@ -270,12 +232,9 @@ func (t *TxThrottler) Close() {
 // should back off). Throttle requires that Open() was previously called
 // successfully.
 func (t *TxThrottler) Throttle() (result bool) {
-	if !t.config.enabled {
+	if !t.config.enabled || t.state == nil {
```

Comment:
Again, I personally prefer the two `if` statements, they provide more clarity IMO.